satpy/tests/compositor_tests/__init__.py

```diff
@@ -1162,6 +1163,100 @@ def test_get_angles(self, get_satpos):
         self.assertEqual(args[6], 0)
 
 
+class TestMaskingCompositor(unittest.TestCase):
+    """Test case for the simple masking compositor."""
+
+    def test_init(self):
+        """Test the initializiation of compositor."""
+        from satpy.composites import MaskingCompositor
+
+        # No transparency given raises ValueError
+        with self.assertRaises(ValueError):
+            comp = MaskingCompositor("name")
+
+        # transparency defined
+        comp = MaskingCompositor("name", transparency=0)
+        self.assertEqual(comp.transparency, 0)
+
+    def test_call(self):
+        """Test call the compositor."""
+        from satpy.composites import MaskingCompositor
+        from satpy.tests.utils import CustomScheduler
+
+        flag_meanings = ['Cloud-free_land', 'Cloud-free_sea']
+        flag_values = np.array([1, 2])
+        transparency_data_v1 = {'Cloud-free_land': 100,
+                                'Cloud-free_sea': 50}
+        transparency_data_v2 = {1: 100,
+                                2: 50}
+
+        # 2D data array
+        data = xr.DataArray(da.random.random((3, 3)), dims=['y', 'x'])
+
+        # 2D CT data array
+        ct_data = [[1, 2, 2],
+                   [2, 1, 2],
+                   [2, 2, 1]]
+        ct_data = xr.DataArray(ct_data, dims=['y', 'x'])
+        ct_data.attrs['flag_meanings'] = flag_meanings
+        ct_data.attrs['flag_values'] = flag_values
+
+        reference_alpha = [[0, 0.5, 0.5],
+                           [0.5, 0, 0.5],
+                           [0.5, 0.5, 0]]
+        reference_alpha = xr.DataArray(reference_alpha, dims=['y', 'x'])
+
+        # Test with numerical transparency data
+        with dask.config.set(scheduler=CustomScheduler(max_computes=0)):
+            comp = MaskingCompositor("name", transparency=transparency_data_v1)
```

Comment:
The data arrays are numpy arrays inside, not dask arrays... so dask won't be involved in this test anyway

Reply:
or at least for `ct_data`

Reply:
Now all the arrays are - not only the input data -  dask arrays, still no computations.